encryption_works.md

```diff
@@ -98,25 +98,58 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
+While your web habits can often be tracked when browsing the Internet, it's still possible to use the web and avoid many types of mass surveillance. The easiest way is to use the Tor Browser, a web browser like Firefox or Safari, but it allows you to browse the web while masking your IP address, which is generally an accurate representation of your approximate location. 
+
+The software is free and open-source, meaning that it comes with a high degree of trust, and is an adapted version of the Firefox browser.
+
+![The Tor Browser](https://raw.githubusercontent.com/tommycollison/encryption-works/master/images/tor_FPF.png)
+
+The Tor network is made up of over 6,000 volunteer servers, called nodes. When you use Tor, your connection is routed through three of these nodes — called a circuit – before exiting onto the normal Internet. Anyone intercepting Internet traffic will think your location is the final node which your traffic exits from.
+
+![To obscure your location, Tor routes your connection through three intermediaries.](/images/tor_relays.png)
+
+### Setup
+
+The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en). Once you download and install it, you'll see that it works like any other web browser but is a little bit slower, since your traffic is being routed through the Tor network to provide you with anonymity.
+
+When Snowden [was answering questions on Guardian's website](http://www.guardian.co.uk/world/2013/jun/17/edward-snowden-nsa-files-whistleblower) from a "secure Internet connection", he was probably routing his traffic through the Tor network. He may have also been using a [bridge](https://bridges.torproject.org/) to connect to the Tor network to make the fact that he was using Tor from his IP address less obvious to eavesdroppers.
+
+### Browsing Habits
+
+Here are some further tips to enhance your security and privacy while using the Tor browser. Some tips, such as disabling flash, are good advice for any web browser.
+
+- Turn on No Scripts for added security, as this prevents javascript from running in the browser, which could put you at risk of being identified.
+- Be aware of unencrypted sites, which don't have a lock icon next to the URL. Tor anonymizes your Internet traffic but unencrypted connections can still be eavesdropped on between the final node and the Internet server.
+- Try to avoid downloading files such as PDFs or Microsoft Word
+documents (which end in .doc or .docx), as they can be vehicles for malware that can be used by an attacker to de-anonymize your web browsing.
+
+As with Tails, these limitations are designed to keep Tor as secure as possible. They can get in the way of your regular browsing habits, and so you should keep another web browser around (with add-ons such as [HTTPS Everywhere](https://www.eff.org/https-everywhere)) for other browsing.
```

Comment:
Sorry, I misread this section. I thought you were encouraging people to add HTTPS-Everywhere to the Tor Browser.

This can stay for now, but we might want to consider expanding this later. There's a lot that can be done to enhance security and privacy in a normal web browser, and recommending people install a couple of add-ons isn't necessarily sufficient.